### PR TITLE
User deletion endpoint

### PR DESCRIPTION
**Pull request by wontheone1**
WIP User deletion endpoint

### CODE REVIEW

File: src/sdk.js [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-import AddGrantTypeToParams from './interceptors/add_grant_type_to_params';
-import AddTokenExchangeGrantTypeToParams from './interceptors/add_token_exchange_grant_type_to_params';
-import AddScopeToParams from './interceptors/add_scope_to_params';
-import AuthInfo from './interceptors/auth_info';
-import MultipartRequest from './interceptors/multipart_request';
-import TransitRequest from './interceptors/transit_request';
-import TransitResponse from './interceptors/transit_response';
-import { createDefaultTokenStore } from './token_store';
-import contextRunner from './context_runner';
+import axios from "axios";
```
Comment (wontheone1) [inline src/sdk.js:1]: @rap1ds @OtterleyW 

I guess my VS code has very different idea of how we should format JS. I see `.eslintrc.js` on the root. Is that enough? I guess I need to somehow make my VS follow that lint config.

Comment (wontheone1) [reply src/sdk.js]: yarn format can fix styling issue.

**Review by bladealslayer — APPROVED** [review]
lgtm!

**Review by rap1ds — APPROVED** [review]
LGTM! 

(But in case I manage to merge #155 before you, you need to move this to a new `endpoints.js` file)

**Pull request merged**